package.json

```diff
@@ -29,7 +29,9 @@
     "dev": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "start:development": "REACT_APP_NODE_ENV=development && react-scripts start",
```

Comment:
we don't need these two new commands? 
When we run `npm run dev`, it will be `NODE_ENV=development` already. 
and it's `"production"` on Heroku.